core/src/main/java/io/questdb/cutlass/Services.java

```diff
@@ -87,7 +87,7 @@ public static HttpServer createHttpServer(
             return null;
         }
 
-        final HttpServer server = new HttpServer(configuration, cairoEngine.getMessageBus(), metrics, workerPool);
+        final HttpServer server = new HttpServer(configuration, cairoEngine.getMessageBus(), metrics, workerPool, configuration.getFactoryProvider().getHttpSocketFactory());
```

Comment:
this is formatting i am constantly undoing, would you mind changing?

it is either this, if call fits a line:

```java
void func(int a, int b, int c);
```
or 

```java
void func(
   int a,
   int b,
   int c
)
```

Not good:

```java
void func(int a, int b
   int c, int d);
```

Reply:
ooops, GH rendering issue, code is fine